examples/mobile/UIComponents/js/app.js

```diff
@@ -19,4 +21,19 @@
 			}
 		}
 	});
+
+	document.addEventListener("pagebeforeshow", function () {
+		var themeChanger = document.querySelector("#theme-selector"),
+			themeChangerButton = document.querySelector("#selector-opener");
+
+		themeChanger.addEventListener("change", function (event) {
+			tau.theme.setTheme(event.target.value);
+		});
+
+		themeChangerButton.addEventListener("click", function () {
+			var dropDownMenuWidget = tau.widget.DropdownMenu(themeChanger);
+
+			dropDownMenuWidget.open();
+		});
+	});
```

Comment:
Need to remove the event listeners when "pagebeforehide".

Reply:
done